src/components/Header/Header.module.scss

```diff
@@ -1,6 +1,145 @@
 @import "../../styles/var.scss";
 
 .header {
-	font-size: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+  background-color: #f8f9fa;
+  font-size: 16px;
+}
+
+.container {
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.logoSection {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+  margin: 10px 0;
+}
+
+.logo img {
+  max-width: 150px;
+}
+
+.searchInput {
+  padding: 8px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  width: 450px;
+  font-size: 16px;
+}
+
+button {
+  border: none;
+  background: none;
+  cursor: pointer;
+  margin: 0 10px;
+}
+
+/*SingInForm */
+.signInContainer {
+  height: 640px;
+  max-width: 980px;
```

Comment:
absolute values for these must be inside media queries
otherwise, please use relative units, e.g.: rem, em, %